src/components/namelist/nameList.css

```diff
@@ -0,0 +1,10 @@
+.create-post-user-details, .student-view {
+    display: grid;
+    grid-template-columns: 50px 1fr 50px;
+    column-gap: 20px;
+    padding:2px;
+
+}
+.post-user-name{
+    padding-top:15px
+}
```

Comment:
It would be nice to tighten up on code-style consistency please - some whitespaces are missing (lines 5, 8 and 9) and there's an unnecessary new line at line 6.